fossweeks/index.html

```diff
@@ -0,0 +1,13 @@
+---
```

Comment:
Please create a tag "fossweeks" and use it here. similar to other pages so that we could view all the fossweeks related posts in a page. 

Have a look at testing related posts to find the usage.

Reply:
I've updated the requested changes. And tested it by adding a blog to that tag. Kindly review it now. And let me know how it looks.